### ISSUE TITLE
Provide an option to disable opensearch serverless

### ISSUE THREAD

**Issue by kcopsguru** [awslabs/visual-asset-management-system#132]
Opensearch Serverless is expensive. VAMS should provide an option to disable it for development. Also the [costs](https://github.com/awslabs/visual-asset-management-system#costs) section should be updated.

**Comment by Hegemege** [comment]
Agreed. After disabling contributor insights for DynamoDB, the daily idle running cost has been $0.21. After updating to 1.4.0 earlier this week, the cost for a full day of OpenSearch serverless being idle was $23.04.

**Comment by archieco** [comment]
Thanks for the feedback. We are thinking about ways to make this an optional feature.

**Comment by scheurik** [comment]
An option here is provide a configuration switch with the new config system to go back to the previous implementation of returning all results and having the front-end do the filtering (or possibly a lambda). This would be an option for those looking to test or PoC VAMS without the costs of OpenSearch.
